frontend/src/pages/admin/access/ManageAccessRequests.scss

```diff
@@ -1,4 +1,48 @@
+$header-height: 72px;
+$footer-height: 46px;
+$navbar-height: 45px;
+$filter-height: 76px;
+$contentheading-height: 60px;
```

Comment:
Please check the other places we have this. I think Devin moved this to a common scss file that can be included there... @devinleighsmith ?

Reply:
I found them